server/controllers/recommendation/bookToPpudController.ts

```diff
@@ -137,6 +149,37 @@ async function post(req: Request, res: Response, _: NextFunction) {
       indexOffence: recommendation.bookRecallToPpud?.indexOffence,
       dateOfIndexOffence: nomisOffence.offenceDate,
     })
+
+    await ppudUpdateRelease(token, offenderId, sentenceId, {
+      dateOfRelease: nomisOffence.releaseDate,
+      postRelease: {
+        assistantChiefOfficer: {
+          name: acoSigned.createdByUserFullName,
+          faxEmail: acoSigned.emailAddress,
+        },
+        offenderManager: {
+          name:
```

Comment:
Nulls aren't allowed for name, faxEmail, telephone. Empty string is fine.

Reply:
not a problem.